### PR TITLE
Use Platform-specific Native Commands

### LINKED ISSUE #2: Issue - npx create-rubric-app@latest failing in Windows 11 during project bootstrap
**Issue Description**:

While using npx create-rubric-app@latest to bootstrap a new Next.js project on Windows 11, I encountered an error upon setup. I am being prompted with the options to "scaffold project files, download assets, configure vscode, configure DB, run install, run dev," but the process does not complete successfully.

Instead, it stalls at the second step "✅ 1/6 - Scaffolded project files" and throws an error `'cp' is not recognized as an internal or external command`. Below is the detailed error I get: 

```JavaScript
'cp' is not recognized as an internal or external command,
operable program or batch file.
node:internal/errors:866
  const err = new Error(message);
              ^

Error: Command failed: cd todo-1 && cp .env.example .env && echo sk-vp5FlPMkqHMmNLKR2OSzXXXXXX >> .env && mv gitignore .gitignore && git init -b main
    at checkExecSyncError (node:child_process:890:11)
    at Object.execSync (node:child_process:962:15)
    at file:///C:/Users/XXXXX/AppData/Local/npm-cache/_npx/938b4f2cd598e4a3/node_modules/create-rubric-app/index.mjs:203:16
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  status: 1,
  signal: null,
  output: [ null, null, null ],
  pid: 15800,
  stdout: null,
  stderr: null
}
```

I'm using Node.js v20.8.0

This seems to be related to the Windows operating system and its lack of built-in support for Unix-based utilities like 'cp'.

A potential solution might be for the create-rubric-app script to determine the user's operating system and issue the appropriate commands accordingly. For instance, in Windows, the 'cp' utility equivalent could be 'copy' or 'xcopy' and 'mv' could be replaced with 'ren' or 'move'.

### PR DESCRIPTION
**Pull request by arihanv**
Changes to resolve #2 .

### CODE REVIEW

**Comment by arihanv** [comment]
Need to review/merge #13 first.

**Comment by tedspare** [comment]
Awesome work again @arihanv!! Were you able to test on a Windows machine?

As well, #12 is merged, feel free to pull from main 💪

**Comment by arihanv** [comment]
> Awesome work again @arihanv!! Were you able to test on a Windows machine?

Thanks! I wasn't since I use Mac, but the code was short and easy enough to read, so I just manually identified commands that are native only to Mac and added code to support them in windows. Would be good if someone tests it out, perhaps the original author of the issue.

**Comment by DexterStorey** [comment]
Still works on Mac. Dev and Build pass. Will merge to main and ask a windows friend to test latest in prod. LGTM, great work @arihanv!

File: index.mjs [hunk 1]
```diff
@@ -25,6 +25,25 @@ const __dirname = path.dirname(__filename)
...
+			return {
+				copy: 'cp',
+				move: 'mv',
+			}
+		//TODO: add more platforms & relevant commands
+	}
+}
+
+const PLC = getPlatformNativeCmds(process.platform)
+
```
Comment (DexterStorey) [inline index.mjs:46]: LGTM 🚀

**Review by DexterStorey — APPROVED** [review]
approved

**Pull request merged**